__tests__/components/Sidebar.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Sidebar from '../../components/Sidebar';
+import { screen, waitFor, fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+test('sidebar componenet renders', async () => {
+  render(<Sidebar />);
+  let dom = await waitFor(() => screen.getByRole());
```

Comment:
Why console log here? We already have a jest setup so you can test this feature without any blockers.

Reply:
I was still getting started with testing and I didn't know how to make it work, for that reason I was experimenting.